edx_credentials_themes/__init__.py

```diff
@@ -1 +1 @@
-__version__ = '0.3.25'
+__version__ = '0.3.26'
```

Comment:
I don't think we need to bump the version on a non-functional/README update.

Reply:
lol, yeah, I realized this after I shut down my computer. I will go ahead and revert that change.